### PR TITLE
Navbar: Add hover border/outline to profile menu icon

### PR DESCRIPTION
**Pull request by ausnewland11**
## Description

Added outline and border color to the profile menu icon when in a hover state.

References [#2088](https://github.com/trimble-oss/modus-web-components/issues/2088) 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-navbar/modus-navbar.scss [hunk 1]
```diff
@@ -228,6 +224,13 @@ nav {
         > :first-child:active {
           outline: $rem-4px solid $modus-navbar-blue-profile-icon-active-border-color;
         }
+
+        &:hover {
+          cursor: pointer;
```
Comment (cjwinsor) [inline stencil-workspace/src/components/modus-navbar/modus-navbar.scss:229]: Properties should be in alphabetical order, running `npm run lint` in the `stencil-workspace` will resolve this and any other issues.

**Comment by cjwinsor** [comment]
@ausnewland11 I'm not sure this matches what the UX expected. Can you work with your UX to verify this is as intended.

**Review by cjwinsor — CHANGES_REQUESTED** [review]

**Comment by ausnewland11** [comment]
> @ausnewland11 I'm not sure this matches what the UX expected. Can you work with your UX to verify this is as intended.

@cjwinsor I checked with UX and have updated the styling.

**Review by cjwinsor — COMMENTED** [review]
I'm currently in the middle of reviewing.  I want to validate with UX that the Focus and Hover+Active states are correct.
image

What I will be checking next:

- Light and Dark themes (I know you didn't update, but we should be for consistency)
- With and without avatar image (I only tested with initials so far)

**Comment by cjwinsor** [comment]
@ausnewland11 For the active+hover state, i'm expecting it not to have an extra thick border and to match active state.  Also can you apply similar changes to light and dark themes so they aren't left out and everything is consistent.

**Comment by DafneeZambrano** [comment]
@cjwinsor @enowak1031 @egunther39 I brought this to the UX team. We are ok with the focus, hover, and active states. We do not need "Active + hover" as the active state does not need to change on hover. It should remain active until the user exits the menu. We also do not need "Tabbed into" as it serves the same purpose as Focus. We do not want to overcomplicate the states. Ewa if "active + hover" and "Tabbed into" are needed for other teams, then we will leave it up to Modus to decide.

**Comment by cjwinsor** [comment]
> @cjwinsor @enowak1031 @egunther39 I brought this to the UX team. We are ok with the focus, hover, and active states. We do not need "Active + hover" as the active state does not need to change on hover. It should remain active until the user exits the menu. We also do not need "Tabbed into" as it serves the same purpose as Focus. We do not want to overcomplicate the states. Ewa if "active + hover" and "Tabbed into" are needed for other teams, then we will leave it up to Modus to decide.

@DafneeZambrano, to clarify.  The active + hover state is caused by a user hovering over the element and then interacting with it with a mouse click.  In that moment the styling for active is being combined with the style for hover.  At the point the mouse click starts it should be active and override the hover. From a UX perspective there is no need to detail out this interaction as it makes sense that active should override the hover style.  

As for tabbed into, this was just to illustrate the styling, which in this case was as expected. I think I might have messed up my screenshot for the notification icon, as doing that test again has it match focus... which the original difference was what lead me to include it in the first place.

**Review by cjwinsor — APPROVED** [review]

**Pull request merged**